src/Console/Command/DescribeCommand.php

```diff
@@ -144,8 +146,18 @@ private function describeRule(OutputInterface $output, $name)
             $definition = new FixerDefinition('Description is not available.', []);
         }
 
+        $description = $definition->getSummary();
+        if ($fixer instanceof DeprecatedFixerInterface) {
+            $successors = $fixer->getSuccessorsNames();
+            $message = null === $successors
+                ? 'not supported anymore'
```

Comment:
This is not true: a deprecated fixer is still supported. The message should rather say that the fixer will be removed on next major version.